imports/plugins/included/payments-stripe-connect/server/Methods/methods.js

```diff
@@ -0,0 +1,27 @@
+import { Meteor } from "meteor/meteor";
```

Comment:
That's because git is case insensitive. Have a look see 
http://stackoverflow.com/questions/10523849/changing-capitalization-of-filenames-in-git